packages/tabs/src/component-tabs.tsx

```diff
@@ -1,9 +1,10 @@
-import React, {
```

Comment:
When this is removed, TS complains about all returned jsx elements: 
```
'React' refers to a UMD global, but the current file is a module. Consider adding an import instead. ts(2686) 
```
Also, the `pnpm build` fails.